### DISCUSSION TITLE
Gordian Developer Meeting in September (EU Friendly Time)

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Gordian-Developer-Community#115] [General & Announcements]
This is a reminder that our upcoming Gordian Developer Meeting for September will be held at a European-friendly time: on Wednesday September 5th at 10am PDT, which is 1pm EDT and 7pm CET.

We'd love to hear you talk about your work! Let us know if you'd like to make a brief presentation of 5 or so minutes about your current work, current questions, or other topics of interest. Just reply to this post or add a new topic in this discussion area and we'll add you to the agenda.

Some things that we're considering talking about so far are:

- Gordian at IETF - updates to our internet-draft and reference code based on IETF F2F in July.

We hope to see you there!

The Blockchain Commons Team

**Comment by OR13** [comment]
I've been working on a YAML format for disclosing documents in JSON and hopefully soon in CBOR.

The approach taken is basically identical to what you see here: https://github.com/danielfett/sd-jwt/blob/main/tests/testcases/array_recursive_sd_some_disclosed/specification.yml

But I am hoping to generalize it away from just working with JSON.

I hope to be far enough along to share more in the meeting, i'm interested in if it can be applied to gordian in addition to sd-jwt, and if not are there learnings that would be relevant to applying it to sd-cwt, or similar cbor structures in the future.

**Reply by shannona** [reply]
Thanks! I've pointed Christopher this way to take a look and see what he thinks.

**Reply by ChristopherA** [reply]
@OR13 if you can be ready for next week’s call, we’d be glad to add you to agenda!

**Reply by OR13** [reply]
I can be ready, I have a pseudo prototype in a Branch

**Reply by OR13** [reply]
Here is an example with a payload that is JSON: https://github.com/w3c/vc-jose-cose-test-suite/blob/main/testcases/secured-vc-jwt-sd/spec.yaml

the idea is to just convert the yaml to CBOR instead of JSON, but otherwise keep using the `!sd` tag in yaml to indicate the data should be "disclosable".

I'm not sure if any of this is useful to Gordion, but happy to help find that out.

**Reply by shannona** [reply]
All sounds terrific. I've got you on the agenda, @OR13. Hopefully you already have a Zoom link from our email last week, but if not just be sure you're on our Gordian Developers Signal channel and Christopher will send it there just before the meeting.

https://www.blockchaincommons.com/subscribe.html

It's 10am PDT tomorrow (Wednesday).

**Comment by shannona** [comment]
A PREVIEW OF NEXT WEDNESDAY'S MEETING

Interoperability is a crucial element to ensure a self-sovereign ecosystem of digital-asset ownership, and we need your feedback to get it right! We've recently added "attachments" to Gordian Envelope, to allow vendors to incorporate their own specific, typed information. You can find the details in our newest Research article:
https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2023-006-envelope-attachment.md

We expect this will be used to store descriptors, sharded shares, backup words, and other specific content. The privacy-preserving elements of Gordian Envelope allow for the safe storage of this sort of sensitive data. But, we need to know what else you might be storing (and how) so that we can help you to produce interoperable specifications for your content.

This is of course part of our Collaborative Seed Recovery system. We look forward to working with you all to develop a system that's much better than that proposed by Ledger earlier this year, due to our shared interest in not just resilience, but also independence, privacy, and openness.

So please JOIN US at September's Gordian Developer Meeting on Wednesday September 6, 2023 at 10am PDT to learn about Envelope's new attachments for CSR and to talk about your needs for interoperable data storage and transmission.

As usual, we'll be at our typical Zoom address (see Signal or Gordian Developer emails or send one of us a PM).

Other topics will include a report out on IETF 117 and how it's affected Gordian Envelope and a look at our new Developer website. If you've had any recent advances, innovations, or inspirations, please let us know, and we'll add that to the schedule. We'll see you there!
